### ISSUE TITLE
Highlight (color) profitable items in club based on Bought For - Futbin price

### ISSUE THREAD

**Issue by kashefcom** [Mardaneus86/futwebapp-tampermonkey#122]
Expected behavior:
When browsing our Club -> Players Items, rows will be colored based on potential profitability. Profitability is calculated as ("Futbin Price" - "Bought For Price") / "Bought For Price" (while +ve).

Current behavior:
No coloring is applied at the moment, only futbin prices are shown for each item

- v1.3.0
- Chrome
- Win7

**Comment by Feigenstaub** [comment]
ouh that's interesting, could maybe be made aswell for just browsing through the market?

**Comment by kashefcom** [comment]
I believe you want to compare futbin price with bin price, an enhancement request has already been opened for that.

**Comment by Regaddi** [comment]
@kashefcom
You're only referring to the club players view, correct?
How should we handle players, that haven't been bought e.g. from pack openings? IMO they shouldn't get this highlighting.

**Comment by kashefcom** [comment]
Yes, no highlighting in that case since they're pack pulled. I would've suggested a neutral color to indicate pack pulled players but since it seems we will have a lot of colored rows I would refrain to avoid any clashes across the app.

**Comment by Mardaneus86** [comment]
As I think about it I'd like to refrain from using a lot of colors throughout. It is most useful to the general public for comparing transferlist auctions to Futbin BIN to find "bargains". In all other cases there isn't a time constraint so it may be better to use some sort of icons?

Since the project is about enhancing the UX of the web app, we shouldn't make it worse 😆

**Comment by kashefcom** [comment]
An icon would be absolutely fine as well, maybe a money icon with a small overlay of the profit alongside it? I suggested color highlighting based on the previous requests and wasn't worried about it clashing since the club players view is separate from the transfer market.

**Comment by kashefcom** [comment]
Something along those lines possibly?

![image](https://user-images.githubusercontent.com/21028224/46915650-848ea480-cfae-11e8-84ad-15426a9a3684.png)
